### PR TITLE
Configure CI to cancel builds when PRs are updated.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: devel/ci/githubprb-project.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     org-list:
       - fedora-infra
     failure-comment: 'This pull request fails CI testing. Please review the Jenkins job. Hint: You can search for "JENKIES FAIL" in the console output to quickly find errors.'
+    cancel-builds-on-update: true
```
Comment (cverna) [inline devel/ci/githubprb-project.yml:27]: Does that cancel current running build and then triggers a new one ?

Comment (bowlofeggs) [reply devel/ci/githubprb-project.yml:27]: @cverna I assume so, but all I have to go on are these docs which do not answer that question:

https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request

Comment (cverna) [reply devel/ci/githubprb-project.yml:27]: ok :) let's try it then

**Comment by cverna** [comment]
:+1:

**Review by cverna — DISMISSED** [review]

**Comment by bowlofeggs** [comment]
I'm going to manually rebase and merge this one.

**Pull request merged**